TaskHouseApi/Model/Worker.cs

```diff
@@ -0,0 +1,12 @@
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace TaskHouseApi.Model
+{
+
```

Comment:
remove newline

Reply:
kan ikke se på de andre dokumenter hvad det er for nogle formatter i vil havde de andre er ikke ens? er det linjen over eller under?